workspaces/topology/plugins/topology/app-config.backstage-community.yaml

```diff
@@ -0,0 +1,14 @@
+dynamicPlugins:
+  frontend:
+    janus-idp.backstage-plugin-topology:
```

Comment:
Should this be `backstage-community.plugin-topology`?

Reply:
Good catch, thanks for the review.